app/components/task/holder.hbs

```diff
@@ -0,0 +1,31 @@
+<div class="task-card">
+  <Task-Details @task={{@task}} />
+  <div class="status-container">
+    {{#if @allowChangePercentage}}
+      <select name="percentage-complete" id="percentCompleted" {{on "change" this.onPercentageChange}}>
+        <option value="">Percentage Completed</option>
+        <option value="20">20</option>
+        <option value="40">40</option>
+        <option value="60">60</option>
+        <option value="80">80</option>
+      </select>
+    {{/if}}
+    <select name="task-status" id="status" {{on "change" this.onStatusChange}}>
+      <option value="">Select Status</option>
+      <option value="active">Active</option>
+      <option value="pending">Pending</option>
+      <option value="blocked">Blocked</option>
+      <option value="completed">Completed</option>
+    </select>
+  </div>
+  <div class="update-task__btn">
+    <button type="button" id={{@task.id}} {{on "click" @onSubmit}}>Update Task</button>
+  </div>
+  <div class="progress-bar__container">
+    <div class="progress-bar">
+      <div {{! template-lint-disable }} class="bar" data-size="40" style="width: {{@task.percentCompleted}}%;">
```

Comment:
Reason for `{{! template-lint-disable }}` :

![image](https://user-images.githubusercontent.com/56217868/109398716-7154b780-7964-11eb-8ace-c6d4393496d4.png)

Reply:
Why are we not fixing the problem the right way. I don't like how this is disabled to avoid a potential reason why the problem existed in the first place

Reply:
https://deprecations.emberjs.com/v1.x/#toc_binding-style-attributes
Will read this and the terms mentioned here for better understanding 👍🏻

Reply:
Removed inline CSS